string joint_name       # Name of the joint (specified in the yaml file)
float64 position        # Position to move to in radians
float32 speed           # Speed to move at (0.0 - 1.0)
float32 acceleration    # Acceleration to move at (0.0 - 1.0)
